Gravity-tube/src/Components/VideoCard/VideoCard.js

```diff
@@ -55,8 +66,10 @@ function VideoCard({ video }) {
             </div>
           </div>
 
+
+
           {showOptions && (
-            <div ref={domNode} className="more-option-box">
+            <div ref={domRef} className="more-option-box">
               <div className="options">
               {videoDataState.watchLater.find((item)=>item._id === video._id) ? (
```

Comment:
As this line of code is repeating in multiple files you can create a utility function for it. This way you won't have to write it  multiple times.